templates/config.edn

```diff
@@ -146,33 +169,38 @@
  ;;  :editor/right           ["ctrl+l" "right"]}
  :shortcuts {}
 
- ;; By default, pressing `Enter` in the document mode will create a new line.
- ;; Set this to `true` so that it's the same behaviour as the usual outliner mode.
- :shortcut/doc-mode-enter-for-new-block? false
+ ;; Configure the behavior of pressing Enter in document mode.
+ ;; if set to true, pressing Enter will create a new block.
+ ;; Default value: true
+ :shortcut/doc-mode-enter-for-new-block? true
 
- ;; Block content larger than `block/content-max-length` will not be searchable
- ;; or editable for performance.
+ ;; Limit the searchable and editable block content length for performance
+ ;; Default value: 10000
  :block/content-max-length 10000
 
- ;; Whether to show command doc on hover
+ ;; Display command documentation on hover.
+ ;; Default value: true
  :ui/show-command-doc? true
```

Comment:
How do I activate this to see it? Would be good to add a comment saying how to, as well.